evaluation/evaluator.go

```diff
@@ -200,7 +200,7 @@ func (e Evaluator) evaluateVariationMap(variationsMap []rest.VariationMap, targe
 
 func (e Evaluator) evaluateFlag(fc rest.FeatureConfig, target *Target) (rest.Variation, error) {
 	var variation = fc.OffVariation
-	if fc.State == rest.FeatureState_on {
+	if fc.State == rest.On {
```

Comment:
Is this changing from `FeatureState_on` to `On` something that we can control?

Reply:
Yeah i've just ran some backwards compatibility stuff i've found and been able to revert this